src/SwedbankPay.Sdk/PaymentInstruments/MobilePay/IMobilePayPayment.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+
+namespace SwedbankPay.Sdk.PaymentInstruments.MobilePay
+{
+    public interface IMobilePayPayment
+    {
+        /// <summary>
+        /// The relative URI and unique identifier of the capture resource this capture belongs to.
+        /// </summary>
+        Uri Id { get; }

```

Comment:
Just a thought: Could many of the properties of the different `IXyzPayment` interfaces be extracted to a common `IPayment` interface?

Reply:
Probably? They share a few common properties, but will we gain anything from it?

Reply:
If we manage to extract an interface with a few common properties, we may implement that interface with a base class all other `XyzPayment` classes can inherit, making the code more DRY.